src/main/java/com/example/demo/model/City.java

```diff
@@ -0,0 +1,42 @@
+package com.example.demo.model;
+
+import org.springframework.stereotype.Component;
+
+@Component
```

Comment:
For beans @component should not be needed as we don't want to inject them and also we don't intend to use them singleton, please check